runtime/doc/vim_diff.txt

```diff
@@ -151,6 +151,7 @@ Commands:
   |:sign-define| accepts a `numhl` argument, to highlight the line number
 
 Events:
+  |CursorMoved| also triggers when moving between windows
```

Comment:
This is a bit misleading, in vim it fires _most of the time_ when switching window, except when specifically the cursor position happens to match up perfectly.

Reply:
```suggestion
  |CursorMoved| always triggers when moving between windows
```

Reply:
This should also be in the "Changed features" section.